test/unit/adapters/catalyst-client.spec.ts

```diff
@@ -29,96 +31,103 @@ jest.mock('../../../src/utils/timer', () => ({
   sleep: jest.fn()
 }))
 
-const CATALYST_CONTENT_LOAD_BALANCER_URL = 'http://catalyst-server.com/content'
+const CATALYST_LAMBDAS_LOAD_BALANCER_URL = 'http://catalyst-server.com/lambdas'
 
 describe('Catalyst client', () => {
   let catalystClient: ICatalystClientComponent
-  let contentClientMock: ContentClient
+  let lambdasClientMock: LambdasClient
 
   beforeEach(async () => {
-    mockConfig.requireString.mockResolvedValue(CATALYST_CONTENT_LOAD_BALANCER_URL)
+    mockConfig.requireString.mockResolvedValue(CATALYST_LAMBDAS_LOAD_BALANCER_URL)
```

Comment:
Just a heads-up, this is working because we are mocking directly `requireString` but I think I saw that the env var name wasn't change on the real implementation.

For future config mocks maybe it is worth placing them at `test/components.ts` directly while overlapping the initComponents.config so you can detect this kind of missmatch in tests 👼